pkg/virtctl/create/create.go

```diff
@@ -21,9 +21,9 @@ package create
 
 import (
 	"github.com/spf13/cobra"
```

Comment:
newline?

Reply:
Basically the same as the other comment: First stdlib imports, then remote packages, then local packages. I did this on purpose while touching the imports anyways.